addons/project/models/project.py

```diff
@@ -1277,6 +1280,31 @@ def _compute_is_private(self):
         for task in self:
             task.is_private = not task.project_id and not task.parent_id
 
+    def _search_is_private(self, operator, value):
+        if not isinstance(value, bool):
+            raise ValueError(_('Value should be True or False (not %s)'), value)
+        if operator not in ['=', '!=']:
+            raise NotImplementedError(_('Operation should be = or != (not %s)'), value)
+        if (operator == '=' and value) or (operator == '!=' and not value):
+            return [('project_id', '=', False)]
+        else:
+            return [('project_id', '!=', False)]
+
+    @api.depends('parent_id.ancestor_id')
+    def _compute_ancestor_id(self):
+        for task in self:
+            task.ancestor_id = task.parent_id.ancestor_id or task.parent_id
```

Comment:
This method is already defined below. It seems you didn't correctly revolve the conflicts when you rebased your branches. ^^